include/openssl/ec.h

```diff
@@ -269,6 +269,13 @@ OPENSSL_EXPORT int EC_POINT_point2cbb(CBB *out, const EC_GROUP *group,
                                       point_conversion_form_t form,
                                       BN_CTX *ctx);
 
+// EC_POINT_point2bn converts an |EC_POINT| to |BIGNUM.| Returns the pointer to the
+// BIGNUM supplied or NULL on error.
+AWS_LC_DEPRECATED OPENSSL_EXPORT BIGNUM *EC_POINT_point2bn(const EC_GROUP *group,
+                                         const EC_POINT *point,
+                                         point_conversion_form_t form,
+                                         BIGNUM *ret, BN_CTX *ctx);
+
```

Comment:
This should go down in the deprecated section around 313. We also still need a test for this function.